### PR TITLE
README-ja2.md

### PR DESCRIPTION
**Pull request by niuni0212**

### CODE REVIEW

**Comment by cmplstofB** [comment]
Why did you [change](https://github.com/HajoRijgersberg/OM/pull/19/commits/ad3a27659cdbacc2e76a673a5ecb28d66a60632e#diff-4c15cb023d73eb32f305098131af8fe9R58) the translation for “we have the following triples” from “次のトリプルで表せる” to “次のように記述される”?
I think that previous translation (even if it is a bit unnatural) represents the original meaning more accurately than new one.

**Comment by cmplstofB** [comment]
Thank you for your explanation. I understand your intention.

I don't think there is a problem with the translation “triple” in this context.
Although it is not clearly defined by official documents such as Japanese Industrial Standards, “triple” as one of the RDF display forms seems to be translated into Japanese as “トリプル”.
For example, in [this document](http://www.soumu.go.jp/main_content/000317915.pdf) published by MIC, the word “トリプル” is used as a translation of “triple”.

In the [informal definition](http://www.asahi-net.or.jp/~ax2s-kmtn/internet/rdf/rdf-concepts.html#section-triples), “RDF triple” is clearly translated as “RDFトリプル”.

**Comment by niuni0212** [comment]
Thank you for your answer and I didn't know it.
According to document you introduced, it seems the word,"トリプル", is general word and I agree to your translation.

File: README-ja.md [hunk 1]
```diff
@@ -44,18 +44,18 @@ OMは次の*応用分野*に対応している:
...
 * 味感・食感の力学
-* 梱包
+* 包装(梱包)
 
 ![OMのUML図](images/OM2.0-UML-diagram.png)
 
 **図1.** OMのクラス構造を表わすUML図
 
-例えば，2016年11月30日午前11時28分における[カナダ・アラート](https://ja.wikipedia.org/?curid=1642577)〔訳注: 最北端の定住地〕の気温は次のトリプルで表せる:
+例えば，2016年11月30日午前11時28分における[カナダ・アラート](https://ja.wikipedia.org/?curid=1642577)〔訳注: 最北端の定住地〕の気温は次のように記述される:
```
Comment (cmplstofB) [inline README-ja.md]: Hi.
Would you please revise “次のように記述される” to the [agreed](https://github.com/HajoRijgersberg/OM/pull/19#issuecomment-530310901) translation “次のトリプルで記述される”?

**Comment by HajoRijgersberg** [comment]
Thank you both so much for all your effort! I appreciate this very much! :)

**Pull request merged**